cypress/plugins/index.js

```diff
@@ -7,6 +7,7 @@
  * @type {Cypress.PluginConfig}
  */
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 module.exports = ( on, config ) => {
```

Comment:
As I understand it, that typescript-eslint rule has the same option as the vanilla eslint rule: https://eslint.org/docs/rules/no-unused-vars

So unless we configured it away somewhere, it should be enough to change the parameters names to start with underscore for that rule to not trigger.

Reply:
Good point, done.